### PR TITLE
Feature/update availability screen

### PR DESCRIPTION
**Pull request by Puththiran**

### CODE REVIEW

File: book-a-reading-room-visit.web/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -15,7 +15,7 @@
             <div class="row">
                 <div class="col-md-12">
                     <p>
-                        Use this service to book a visit to view our documents. You may book a maximum of three visits in 4 weeks.<br>
+                        Use this service to book a visit to view our documents. You may book a maximum of four visits in 4 weeks.<br>
                         <a class="info-link" href="/book-a-reading-room-visit/about-the-book-a-reading-room-visit-service/">
```
Comment (Brian-O-TNA) [inline book-a-reading-room-visit.web/Views/Home/Index.cshtml:19]: Any reason why we're using a mixture of words and numerals for numbers here? i.e. why not "three visits in four weeks" or "3 visits in 4 weeks"

File: book-a-reading-room-visit.web/Views/Shared/AboutTheBookAReadingRoomVisitService.cshtml [hunk 2]
```diff
@@ -38,8 +38,8 @@
                 <h2>Terms of booking</h2>
                 <p>New slots for booking will be made available every day.</p>
                 <p>Each booking is for one person only on a first-come, first-served basis – you will not be able to bring anyone with you, unless they book a visit themselves or have a duty of care.</p>
-                <p>There is a reasonable limit of three bookings of any type (standard or bulk) in a rolling 4-week period. Three bookings cannot be made for the same date.</p>
-                <p>Please be thoughtful and considerate of others when you book – we may cancel your booking if you exceed our reasonable limit of three bookings in 4 weeks.</p>
```
Comment (Brian-O-TNA) [inline book-a-reading-room-visit.web/Views/Shared/AboutTheBookAReadingRoomVisitService.cshtml:42]: Mixture of numeral and words for numbers again.

**Review by Brian-O-TNA — COMMENTED** [review]
Just some editorial points about representing numbers.

File: book-a-reading-room-visit.web/Views/Home/Index.cshtml (hunk 1 above)
Comment (Puththiran) [reply book-a-reading-room-visit.web/Views/Home/Index.cshtml:19]: These are editorial change given by editors

File: book-a-reading-room-visit.web/Views/Shared/AboutTheBookAReadingRoomVisitService.cshtml (hunk 2 above)
Comment (Puththiran) [reply book-a-reading-room-visit.web/Views/Shared/AboutTheBookAReadingRoomVisitService.cshtml:42]: These are editorial change given by editors

**Pull request merged**